src/features/rewards/mobile/settingsPageMobile/style.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  background: ${p => p.theme.color.subtleBackground};
+  min-height: 100vh;
+  width: 100%;
+  padding-top: 35px;
+  font-family: "Poppins", sans-serif
```

Comment:
Nit: another font family that can use theme prop

Reply:
Will update